tests/ApplicationAvailabilityTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Tests;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class ApplicationAvailabilityTest extends WebTestCase
+{
+    /**
+     * @dataProvider urlProvider
+     */
+    public function testPageIsSuccessful($url)
+    {
+        $client = self::createClient();
+        $client->request('GET', $url);
+
+        $this->assertTrue($client->getResponse()->isSuccessful());
+    }
+
+    public function urlProvider()
```

Comment:
@Rabot18 smoke test of URL, it just ping them and check if we get a 200 response code